Languages/ja.ini

```diff
@@ -738,12 +743,12 @@ Fail=Fail
 Restart=Restart Song
 
 # GameplayExtras
-ColumnFlashOnMiss=Flash Column for Miss
-SubtractiveScoring=Subtractive Scoring
-Pacemaker=Pacemaker
-MissBecauseHeld=Track Held Misses
-NPSGraphAtTop=Density Graph at Top
-JudgmentTilt=Judgment Tilt
+ColumnFlashOnMiss=ミスした列を光らせる
+SubtractiveScoring=減算スコア
+Pacemaker=ペースメーカー
+MissBecauseHeld=ホールドNGを記録
```

Comment:
This is a little different. This option tracks whether you got a miss because the panel was held down or not, and not for the hold NG.

ちょっと違います。「ミス」がある時、パネルを押さえたかどうか教えます。
例えば、この数：
![image](https://user-images.githubusercontent.com/5017202/162925406-a3331dc2-3032-475a-bee3-f6d17c642d13.png)

Reply:
Yup!

Reply:
How do you think about this?

```suggestion
MissBecauseHeld=押しっぱなしミスを記録
```

Reply:
いいと思う！